usage/rest-server/src/test/java/brooklyn/rest/resources/ServerResourceTest.java

```diff
@@ -97,5 +99,15 @@ public void testShutdown() throws Exception {
                 assertFalse(getManagementContext().isRunning());
             }});
     }
+
+    @Test
+    void testGetConfig() throws Exception {
```

Comment:
Worth including a test of behaviour if you ask for a config key that is not there.